net-vpn/i2p/files/DataHelper-decompress-fix-length-check.patch

```diff
@@ -0,0 +1,24 @@
+From ceb0749e0d5c3c496b4a91bc3a9d3b5e296fc456 Mon Sep 17 00:00:00 2001
+From: zzz <zzz@mail.i2p>
```

Comment:
Also rename the patch file to be i2p-0.9.47-..., then refer to it as ${P} in the ebuild as it is upstreamed and should be in the next release.

Reply:
Good idea, mv'ed